airflow/contrib/hooks/gcp_dlp_hook.py

```diff
@@ -0,0 +1,1362 @@
+            raise AirflowException("Please provide the parent resource name.")
+
+        return client.create_deidentify_template(
+            parent=parent,
+            deidentify_template=deidentify_template,
+            template_id=template_id,
+            retry=retry,
+            timeout=timeout,
+            metadata=metadata,
+        )
+
+    @GoogleCloudBaseHook.catch_http_exception
+    def create_dlp_job(
+        self,
+        parent,
+        inspect_job=None,
+        risk_job=None,
+        job_id=None,
+        retry=None,
+        timeout=None,
+        metadata=None,
+        wait_until_finished=True,
+        polling_interval_in_seconds=60
+    ):
+        """
+        Creates a new job to inspect storage or calculate risk metrics.
+
+        :param parent: The parent resource name.
+        :type parent: str
+        :param inspect_job: (Optional) The configuration for the inspect job.
+        :type inspect_job: dict or google.cloud.dlp_v2.types.InspectJobConfig
+        :param risk_job: (Optional) The configuration for the risk job.
+        :type risk_job: dict or google.cloud.dlp_v2.types.RiskAnalysisJobConfig
+        :param job_id: (Optional) The job ID.
+        :type job_id: str
+        :param retry: (Optional) A retry object used to retry requests.
+            If None is specified, requests will not be retried.
+        :type retry: google.api_core.retry.Retry
+        :param timeout: (Optional) The amount of time, in seconds, to wait for the request
+            to complete. Note that if retry is specified, the timeout applies to each
+            individual attempt.
+        :type timeout: float
+        :param metadata: (Optional) Additional metadata that is provided to the method.
+        :type metadata: sequence[tuple[str, str]]]
+        :param wait_until_finished: (Optional) If true, it will keep polling the job state
+            until it is set to DONE.
+        :type wait_until_finished: bool
+        :param polling_interval_in_seconds: (Optional) The number of seconds to wait
+            before calling another request. This field will only be used when
+            wait_until_finished is set to True.
+        :type polling_interval_in_seconds: int
+        :rtype: google.cloud.dlp_v2.types.DlpJob
+        """
+
+        client = self.get_conn()
+
+        if not parent:
+            raise AirflowException("Please provide the parent resource name.")
+
+        job = client.create_dlp_job(
+            parent=parent,
+            inspect_job=inspect_job,
+            risk_job=risk_job,
+            job_id=job_id,
+            retry=retry,
+            timeout=timeout,
+            metadata=metadata,
+        )
+
+        while wait_until_finished:
+            job = self.get_dlp_job(name=job.name)
+            state_message = 'DLP job {} state: {}.'.format(
+                job.name,
+                DlpJob.JobState.Name(job.state)
+            )
+
+            self.log.info(state_message)
```

Comment:
```suggestion
            self.log.info(state_message)
```
I think, you should avoid formatting string before passing to logger. 
Reference: 
https://github.com/apache/airflow/pull/4804
https://github.com/apache/airflow/pull/5466